ledger/metrics/src/test/lib/scala/com/daml/metrics/api/testing/MetricValues.scala

```diff
@@ -93,17 +103,37 @@ trait MetricValues {
 
     def getCount: Long = timer match {
       case DropwizardTimer(_, metric) => metric.getCount
+      case timer: InMemoryTimer =>
+        singleValueFromContexts(timer.data.values.toMap.view.mapValues(_.size.toLong).toMap)
       case other =>
         throw new IllegalArgumentException(s"Count not supported for $other")
     }
 
     def getCounts: Map[MetricsContext, Long] = timer match {
       case timer: InMemoryTimer =>
-        timer.runTimers.toMap.view.mapValues(_.get().toLong).toMap
+        timer.data.values.toMap.view.mapValues(_.size.toLong).toMap
       case other =>
         throw new IllegalArgumentException(s"Counts not supported for $other")
     }
 
+    def getValues: Seq[Long] = timer match {
+      case timer: InMemoryTimer =>
+        singleValueFromContexts(timer.data.values.toMap)
+      case other =>
+        throw new IllegalArgumentException(s"Count not supported for $other")
+    }
+
+  }
+
+  private def singleValueFromContexts[T](
+      contextWithValues: Map[
+        MetricsContext,
+        T,
+      ]
+  ) = {
+    if (contextWithValues.size == 1)
+      contextWithValues.head._2
+    else throw new IllegalArgumentException("Cannot get value with multi context metrics.")
```

Comment:
Nit: To follow the [style guide](https://docs.scala-lang.org/style/control-structures.html#curly-braces)
```suggestion
    else
      throw new IllegalArgumentException("Cannot get value from multi-context metrics.")
```